app/views/users/registration.html.erb

```diff
@@ -37,6 +37,15 @@
         </div>
       </div>
 
+      <div class="field">
+        <label class="label">Role</label>
+        <div class="control">
+          <div class="select">
+            <%= f.select :category, @categories %>
```

Comment:
```suggestion
            <%= f.select :category, @categories, multiple: true %>
```
I think we want to use `multiple: true`, so that users can be a participant and an event organizer etc.

Reply:
So it actually looks like this is going to involve a bit more work than just adding the option ... and probably would be best handled in another model as an associated collection, if we really need it (rather than modifying the current column to take an array, validating said array, and so on). Happy to do it if there's a need, but my vote for now would be ship as is, and then once the requirements are solidified in segment, circle back.

Reply:
Yeah, that makes sense to me, let's ship as-is and we can revisit if needed.